includes/admin/class-give-settings.php

```diff
@@ -543,6 +543,19 @@ public function give_settings( $active_tab ) {
 							'type' => 'file'
 						),
 						array(
+						    'id'      => 'email_logo_size',

```

Comment:
@philwp It seems the most specific problem that general. It is easy to crop and upload  an image and then assign as email logo.

@DevinWalker What do you think?